packages/components/src/tabs/README.md

```diff
@@ -0,0 +1,237 @@
+# Tabs
+
+Tabs is a collection of React components that combine to render an [ARIA-compliant tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/).
```

Comment:
Let's add an initial paragraph about this component being experimental for the time being (you should be able to copy that from other components' READMEs)